example/opentelemetry/main.py

```diff
@@ -0,0 +1,151 @@
+import asyncio
+import os
+import sys
+from operator import attrgetter
+from random import randint
+from typing import Any
+
+from dotenv import load_dotenv
+from opentelemetry import metrics
+from opentelemetry.exporter.otlp.proto.grpc.metric_exporter import OTLPMetricExporter
```

Comment:
Ah sorry I overlooked this running the example, here we specify `opentelemetry.exporter.otlp.proto.grpc.metric_exporter` but the requirements specify `opentelemetry-exporter-otlp-proto-http`

Should we switch this to the http one or add the grpc one to the requirements?

Reply:
Ah whoop, good catch, updated! I swapped between them a few times testing and guessed I missed updating that 😆